images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -18,7 +31,7 @@ ForEach ($version in $VersionsList)
     if ($version -eq "9.4.3"){ [version]$version = "9.4.2" }
```

Comment:
we should try to drop these versioning workaround

Reply:
Removed.